src/server_manager/ui_components/outline-manual-server-entry.html

```diff
@@ -200,25 +216,28 @@
                   <li>[[localize('gcp-firewall-create-2')]]</li>
                   <li>[[localize('gcp-firewall-create-3')]]</li>
                   <li>[[localize('gcp-firewall-create-4')]]</li>
-                  <li>[[localize('gcp-firewall-create-5')]]</li>
+                  <li>[[localize('gcp-click-create')]]</li>
                 </ol>
               </outline-cloud-instructions-view>
             </div>
             <div class="section-content-instructions">
+              <!-- TODOBEFOREMERGE images -->
```

Comment:
Is this done? If so, please remove the comment.

Reply:
Done in cae793b